contrib/python/api/skydive/wsshell.py

```diff
@@ -74,6 +77,7 @@ def onOpen(self):
 
 
 def main():
+    logging.basicConfig(level=logging.INFO)
```

Comment:
Can you use DEBUG here otherwise we don't get the messages. This is mostly a debugging tool so make sense to have everything in DEBUG anyway.

Reply:
or maybe a new argument...

Reply:
Done on both changes.